task_6.py

```diff
@@ -0,0 +1,9 @@
+#6
+names = ["Tyler Craig", "Nicola Suarez", "Tyler Craig", "Claire Chang", "Suzanne Mejia", "Claire Chang", "Tyler Craig"]
+unique_names = []
+
+for name in names:
+    if name not in unique_names:
+        unique_names.append(name)
+
+print(unique_names)
```

Comment:
as I mentioned at the lesson you should use the default function of list/set/dict/tuple data types, not for each loop,
and no empty line